src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -125,7 +126,7 @@ class NotebookCard extends Component {
   }
 }
 
-const WorkspaceNotebooks = ajaxCaller(wrapWorkspace({
+const WorkspaceNotebooks = ajaxCaller(Globals.globalObserver('notebooksListView')(wrapWorkspace({
```

Comment:
FYI: When the chain of wrappers gets long, we can use `_.flow` to flatten things out.